Sofa/Component/LinearSolver/Iterative/src/sofa/component/linearsolver/iterative/MatrixLinearSolver.h

```diff
@@ -295,14 +295,6 @@ class MatrixLinearSolver<Matrix,Vector,NoThreadManager> : public BaseMatrixLinea
 
 protected:
 
-    using BaseMatrixLinearSolver<Matrix, Vector>::partial_solve;
-
-    /// newPartially solve the system
-    virtual void partial_solve(Matrix& /*M*/, Vector& /*partial_solution*/, Vector& /*sparse_rh*/, ListIndex& /* indices_solution*/, ListIndex& /* indices input */)
```

Comment:
Have you noticed that this is not the same signature than in LinearSolver? Another weirdness...

Reply:
Yes I saw it while searching again who could potentially implements partial_solve, and at the beginning I quickly thought this was just an override of the base class....